pettingzoo/utils/wrappers/pz_to_gymnasium_wrapper.py

```diff
@@ -0,0 +1,78 @@
+from typing import Callable, Dict, Optional, Protocol, TypeVar, Union
+import gymnasium
+from pettingzoo import AECEnv
+
+
+Action = TypeVar("Action")
+Observation = TypeVar("Observation")
+
+ActOther = Callable[[Observation], Action]
+
+
+class PZ2GymnasiumWrapper(gymnasium.Env):
+  """
+  This class wraps a PettingZoo environment (AECEnv) and exposes a simple Gymnasium environment
+  (for single agent).
+  In order to make it work, one needs to provide in the initialization a mechanism to get the actions
+  of all other agents.
+
+  Note that this class is actually a Gymnasium environment.
+  It is not a PettingZoo wrapper and is neigher a Gymnasium one.
+  """
+
+  def __init__(
+    self,
+    pz_env: AECEnv,
+    act_others: Dict[str, ActOther],
+    take_spaces_from: Union[str, None]
+  ):
+    super().__init__()
+    self._pz_env = pz_env
+    self._act_others = act_others
+    take_spaces_from = (
```

Comment:
this variable naming is confusing and it's not immediately clear to me what this does

Reply:
The interface has changed. Hopefully clearer now.